common/scala/src/main/resources/reference.conf

```diff
@@ -3,4 +3,36 @@ whisk.spi{
   MessagingProvider = whisk.connector.kafka.KafkaMessagingProvider
   ContainerFactoryProvider = whisk.core.containerpool.docker.DockerContainerFactoryProvider
   LogStoreProvider = whisk.core.containerpool.logging.DockerLogStoreProvider
-}
\ No newline at end of file
+}
+
+dispatchers {
+# Custom dispatcher for CouchDB Client. Tune as needed.
+  couch-dispatcher {
+      type = Dispatcher
+      executor = "fork-join-executor"
+
+      fork-join-executor {
+          # Min number of threads to cap factor-based parallelism number to
+          parallelism-min = 2
+          # Parallelism (threads) ... ceil(available processors * factor)
+          parallelism-factor = 2.0
+          # Max number of threads to cap factor-based parallelism number to
+          parallelism-max = 10
+      }
+  }
+
+# Custom dispatcher for Kafka client. Tune as needed.
+  kafka-dispatcher {
+      type = Dispatcher
+      executor = "fork-join-executor"
+
+      fork-join-executor {
+          # Min number of threads to cap factor-based parallelism number to
+          parallelism-min = 2
+          # Parallelism (threads) ... ceil(available processors * factor)
+          parallelism-factor = 2.0
+          # Max number of threads to cap factor-based parallelism number to
+          parallelism-max = 10
+      }
+  }
```

Comment:
Are we at risk to starve this one? Kafka producing **can** be blocking and polling for sure is blocking although there should never be more than a couple of polls active.

Reply:
er, yeah. I think I copied this from docs; it looks like the default for Akka's default dispatchers is 64. I'll move it to that and we can always tweak specifically later.